src/main/java/io/managed/services/test/Environment.java

```diff
@@ -184,7 +180,7 @@ public class Environment {
 
     public static final String PROMETHEUS_PUSH_GATEWAY = getOrDefault(PROMETHEUS_PUSH_GATEWAY_ENV, null);
 
-    public static final String STRATOSPHERE_PASSWORD = getOrDefault(STRATOSPHERE_PASSWORD_ENV, null);
+    public static final String STRATOSPHERE_OFFLINE_TOKEN = getOrDefault(STRATOSPHERE_OFFLINE_TOKEN_ENV, null);
     public static final String STRATOSPHERE_SCENARIO_1_USER = getOrDefault(STRATOSPHERE_SCENARIO_1_USER_ENV, null);
     public static final String STRATOSPHERE_SCENARIO_2_USER = getOrDefault(STRATOSPHERE_SCENARIO_2_USER_ENV, null);
     public static final String STRATOSPHERE_SCENARIO_3_USER = getOrDefault(STRATOSPHERE_SCENARIO_3_USER_ENV, null);
```

Comment:
problem here is that these are all users as well, (like PRIMARY_USERNAME), I understand that this was confusing as they all shared the same password.  this would be reasonable replacement. 
```
    private static final String STRATOSPHERE_SCENARIO_1_USER_OFFLINE_TOKEN_ENV  = "STRATOSPHERE_SCENARIO_1_USER_OFFLINE_TOKEN";
    private static final String STRATOSPHERE_SCENARIO_2_USER_OFFLINE_TOKEN_ENV  = "STRATOSPHERE_SCENARIO_2_USER_OFFLINE_TOKEN";
    private static final String STRATOSPHERE_SCENARIO_3_USER_OFFLINE_TOKEN_ENV  = "STRATOSPHERE_SCENARIO_3_USER_OFFLINE_TOKEN";
    private static final String STRATOSPHERE_SCENARIO_4_USER_OFFLINE_TOKEN_ENV  = "STRATOSPHERE_SCENARIO_4_USER_OFFLINE_TOKEN";
```

Reply:
and also later delete unused
```
public static final String STRATOSPHERE_SCENARIO_1_USER = getOrDefault(STRATOSPHERE_SCENARIO_1_USER_ENV, null);
    public static final String STRATOSPHERE_SCENARIO_2_USER = getOrDefault(STRATOSPHERE_SCENARIO_2_USER_ENV, null);
    public static final String STRATOSPHERE_SCENARIO_3_USER = getOrDefault(STRATOSPHERE_SCENARIO_3_USER_ENV, null);
    public static final String STRATOSPHERE_SCENARIO_4_USER = getOrDefau
```

Reply:
resolved